static/docs/tutorial/preparation.md

```diff
@@ -17,11 +17,11 @@ code into it:
 
 ### Expand to learn how to download on Windows
 
-Windows does not ship `wget` utility by default, so you'll need to use
-browser to download `code.zip` or install it from a third party. We recommend
-using [chocolatey](https://chocolatey.org/). First, if you haven't already,
-install chocolatey using [official guide](https://chocolatey.org/install). Then
-install `wget` and `tar` with the following command in the `Command Prompt`:
+Windows does not ship `wget` utility by default, so you'll need to use browser
+to download `code.zip` or install it from a third party. We recommend using
+[chocolatey](https://chocolatey.org/). First, if you haven't already, install
+chocolatey using [official guide](https://chocolatey.org/install). Then install
+`wget` and `tar` with the following command in the `Command Prompt`:
```

Comment:
let's actually change it to just "use your browser". I don't think we need to complicate stuff with choco on Windows.

Reply:
It's also possible to recommend the Linux Subsystem for Windows - this gives a whole Linux environment that's pretty decent.

Reply:
I also thought about suggesting the Linux sub-system when reading this chocolatey suggestion. Should I change it to that then? Changed to just "use your browser" for now.